### ISSUE TITLE
Database created/modified on Linux (Appimage) won't work on Android

### ISSUE THREAD

**Issue by aFunkyBass** [moneymanagerex/android-money-manager-ex#1703]
**Describe the bug**
If you try to open a database created on Ubuntu Linux Appimage (version 1.8.1 beta) on the Android  app (2024-05-09 1032), it appears to be empty and it prompts to create an account, even though they seemingly have the same database version (v. 19). The file resides on a Nextcloud shared folder

**To Reproduce**
1. Create a database on a shared folder using the Linux desktop app
2. Populate some data
3. Open the same database on the Android app
4. No data are shown even if the file is currently syncronized and you're prompted to create an account

**Expected behavior**
The database should present the data previously inserted on the Desktop App. 

**Device Information:**
 - Device: Realme 8 5G
 - OS: Android 11 + Ubuntu Linux 20.04
 - Storage: Nextcloud shared folder
 - Synchronization: Enabled with 5min interval and Sync on Start
 - Encrypted: no
 - App Version: [e.g. 2024.05.09]

Thanks in advance for your help.

**Comment by aFunkyBass** [comment]
I forgot to add that re-opening the DB on Linux, the data are still there.

**Comment by guanlisheng** [comment]
thanks, @aFunkyBass for trying MMEX.

can you just re-open the DB on Android?

and share more detailed steps & setup to re-produce the issue.  seems related with #1673

**Comment by aFunkyBass** [comment]
Hello @guanlisheng no, it's not a matter of Nextcloud sync, because I've created a new DB from the app on the same folder and it worked.

Today I tried to replicate the same setup on my tablet running Android 8, same MMEX version from F-Droid and it worked... So being more than puzzled, I continued experimenting and now it's working as expected, but in the end with so many tries made, I can't really tell what made it work.

I will use it in the next weeks and see what happens. To be honest, after this experience I have a hard time relying on this system to keep all my personal data, but I want to be positive about it and hope that it will continue working in the long run.

I've been using a spreadsheet to keep track of my expenses for more than 30 years, and I was searching for a cross platform solution easier to manage from a mobile device, but still open source and working on Linux, that's why MMEX was just perfect for me...

I'll report my experience in the next days, if you don't mind.

Thank you for your prompt response and for the great job you do mantaining this app.
